src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/entities/camunda/CamundaSendMessageRequest.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.camunda;
+
+import lombok.Data;
+import uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.Builder;
+
+import java.util.Map;
+
+@Data
```

Comment:
Rather not use this Lombok annotation.  It's adding a whole load of code including setters which I don't think we want in most cases as we'd like objects to be immutable.
Also do you need a Builder pattern in this class?  There are only 2 fields so a constructor could suffice?